src/auth/token-verifier.ts

```diff
@@ -144,6 +150,10 @@ export class FirebaseTokenVerifier {
       );
     }
 
+    if (this.useEmulator) {
```

Comment:
This bypasses all the token verification checks. Wouldn't you want to check for timestamps and other standard claims like issuer? A developer might wish to write tests that check for such scenarios.

Reply:
Good idea, re-instated all checks.